src/asmcnc/apps/systemTools_app/screens/calibration/screen_calibration_test.py

```diff
@@ -1696,7 +1696,7 @@ def send_data_for_each_stage(self, stage):
                 self.calibration_db.insert_final_test_statistics(*statistics)
                 return True
             except:
-                Logger.info(traceback.format_exc())
+                Logger.error(traceback.format_exc())
```

Comment:
I think having a consistent format to the exceptions would be nice, and instead of relying on traceback we can use Logger.exception():

`from asmcnc.comms.logging_system.logging_system import Logger

try:
    print(10/0)
except ZeroDivisionError as e:
    Logger.exception("Division by zero")`

Reply:
👍